### DISCUSSION TITLE
GoogleSigninButtonDirective min version 14.0.0

### DISCUSSION THREAD

**Discussion by Abishek-Satyan** [abacritt/angularx-social-login#561] [Q&A]
Cannot create property 'message' on string '/var/www/html/SalesPro/node_modules/@abacritt/angularx-social-login/fesm2015/abacritt-angularx-social-login.mjs: This application depends upon a library published using Angular version 14.0.3, which requires Angular version 14.0.0 or newer to work correctly.
Consider upgrading your application to use a more recent version of Angular.

**Comment by yanivamrami** [comment]
@Abishek-Satyan  which Angular version are you running?
I am running 13.0 and got the same issue, while the compatibility table clearly states it runs on both 13 & 14...

Did you manage to solve it?

Thanks
